main.star

```diff
@@ -102,6 +102,14 @@ def run(plan, args):
         )
     )
 
+    init_env_file_command = "echo '{0}' > /app/.env".format("\n".join(["{0}={1}".format(k, v) for (k, v) in env_vars.items()]))
```

Comment:
An alternative would be to upload file artifact and have it uploaded into the service during the startup, and in the readme we can say that add your autogpt specific env variables in that file. @laurentluce ( though it's not biggy)

Reply:
Would it create env file in the container if `env_vars` is empty?

Reply:
> Would it create env file in the container if `env_vars` is empty?

Yes and the file would be empty.